lib/scnnr/errors.rb

```diff
@@ -34,7 +34,7 @@ def initialize(message, recognition)
     end
   end
 
-  class UnsupportedError < StandardError
+  class UnexpectedError < StandardError
```

Comment:
Could you write this change to `Changed` section in `CHANGELOG.md`?
Because this change may cause `NameError` so I think that it should be treated as public specifications.

Reply:
Thanks, I think so too. 
I've added information on class name changes!

Reply:
Cool, thanks!